src/content/en/2021/jamstack.md

```diff
@@ -242,8 +261,12 @@ All Lighthouse scores are simulated testing data from our crawl. Hence, real-use
   sql_file="median_lighthouse_score.sql"
 ) }}
 
+{# TODO - this figure description should be described more #}
+
 Median performance score for all SSGs across mobile varies. The top 3 SSGs with data can't even pass 40 score. Since they are pretty much popular in top ranking sites, and distributed all around the world, as we saw on data before, we can assumed that they are used across many different devices and networks. We can expect more out-of-the-box improvements like <a hreflang="en" href="https://nextjs.org/docs/basic-features/image-optimization">Next.js image component</a> to help performance.
 
+{# TODO - there needs to be some grammar fix in the paragraph above but I didn't understand it #}
```

Comment:
```suggestion
The median performance score for all SSGs across mobile varies. The top 3 SSGs with by popularity can't even surpass a score of 40. Since they are used in top ranking sites and since users a likely distributed all around the world, we can assumed that they are used across many different devices and networks. We can expect more out-of-the-box improvements like <a hreflang="en" href="https://nextjs.org/docs/basic-features/image-optimization">Next.js image component</a> to help performance.
```